### PR TITLE
Contrast at dark mode and image loading

### PR DESCRIPTION
**Pull request by josanri**
Improved contrast on dark mode for schedules times. For example:
image

Also set logo image as eager loaded.

### CODE REVIEW

File: src/components/Navigation.astro [hunk 1]
```diff
@@ -20,7 +20,8 @@ const featuredSponsors = DATA.sponsors.filter(
         class="brightness-100 dark:invert"
         height="40"
         width="200"
-      />
+        loading="eager"
+        />
```
Comment (davidrojom) [inline src/components/Navigation.astro:24]: not necessary since eager is the default value

**Review by davidrojom — CHANGES_REQUESTED** [review]

File: src/components/Navigation.astro (hunk 1 above)
Comment (josanri) [reply src/components/Navigation.astro:24]: I think it is still loading lazy as it is shown at the HTML. Astro images are set lazy by default: 
image

Comment (davidrojom) [reply src/components/Navigation.astro:24]: good catch

**Comment by davidrojom** [comment]
LGTM

**Pull request merged**